snooty.toml

```diff
@@ -1,6 +1,21 @@
 name = "compass"
+title = "MongoDB Compass"
+
+intersphinx = ["https://docs.mongodb.com/manual/objects.inv"]
+
+toc_landing_pages = ["/install", "/connect", "/instance", "/query/filter", "/query/queries",
+                     "/documents", "/schema", "/aggregation-pipeline-builder"]
 
 [constants]
-version = "1.18"
-current-version = "1.17.0"
 download-page = "`downloads page <https://www.mongodb.com/download-center/compass?tck=docs_compass>`__"
+current-version = "1.20.4"
```

Comment:
[nit] Does this affect what get shows up when you hover over the tab in your browser? We typically don't document specific minor versions, so maybe this should be "1.21" ?

Reply:
I used what you have for the {{current_version}} source constant in conf.py currently. It appears that source constant is used in install.txt for the RHEL install (line 202) YUM command.

Reply:
d'oh I need to update that 😅